internal/exec/resolvable/resolvable.go

```diff
@@ -86,7 +87,7 @@ func (f *Field) Resolve(ctx context.Context, resolver reflect.Value, args interf
 	return wrapResolver(ctx, args)
 }
 
-func (f *Field) Validate(ctx context.Context, args interface{}) []error {
+func (f *Field) Validate(ctx context.Context, rawArgs map[string]interface{}, args interface{}) []error {
```

Comment:
I don't like that we pass both args and rawArgs here. Should the args validator be called before the field validator? This way the field is always called with valid args. 🤔